test/GithubApi.Repositories.test.js

```diff
@@ -0,0 +1,67 @@
+const axios = require('axios');
+const { expect } = require('chai');
+const { StatusCodes } = require('http-status-codes');
+const chai = require('chai');
+const chaiSubset = require('chai-subset');
+const md5 = require('md5');
+const data = require('./data.json');
+
+const url = 'https://api.github.com/users';
+const userName = 'aperdomob';
+
+chai.use(chaiSubset);
+
+describe('Github Api Repositories Test', () => {
+  let responseData;
+  let repoName;
+  it('GET test', async () => {
+    const response = await axios.get(`${url}/${userName}`);
```

Comment:
instead of declaring response object here, you can assign directly to the `userDataResponse` variable (currently named as `responseData`)